velero-2-mixin/panels.libsonnet

```diff
@@ -90,5 +90,175 @@ local utils = commonlib.utils;
+        + g.panel.gauge.queryOptions.withDatasource('prometheus', '${prometheus_datasource}')
+        + g.panel.gauge.panelOptions.withDescription('Success rate of volume snapshots.')
+        + g.panel.gauge.standardOptions.withUnit('percentunit'),
+
+      restoreValidationFailure:
+        commonlib.panels.generic.stat.info.new(
+          'Restore validation failure / $__interval ',
+          targets=[t.restoreValidationFailure],
+          description='Number of failed restore validations'
+        )
+        + stat.options.withGraphMode('area'),
+      lastBackupStatus:
+        commonlib.panels.generic.stat.base.new(
+          'Backup last status',
+          targets=[t.lastBackupStatus],
+          description='Reports the status of the last backup.',
+        )
+        + stat.standardOptions.withUnit('string')
+        + stat.options.withGraphMode('none')
+        + stat.standardOptions.withMappings({
+          type: 'value',
+          options: {
+            '0': {
+              color: 'red',
+              index: 1,
+              text: 'Down',
+            },
+            '1': {
+              color: 'green',
+              index: 0,
+              text: 'Up',
+            },
+          },
+        }),
+
+      backupCount:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Backup count / $__interval ',
+          targets=[t.backupSuccess, t.backupFailure],
+          description=|||
+            Number of failed and succeeded backups.
+          |||
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withDrawStyle('bars'),
+
+      successfulBackups:
+        commonlib.panels.generic.stat.info.new(
+          'Successful backups / $__interval ',
+          targets=[t.succesfulBackups],
+          description='Number of successful backups'
+        )
+        + stat.options.withGraphMode('area'),
+      failedBackups:
+        commonlib.panels.generic.stat.info.new(
+          'Failed backups / $__interval ',
+          targets=[t.failedBackups],
+          description='Number of failed backups.'
+        )
+        + stat.options.withGraphMode('area'),
+
+      backupSuccessRateTimeseries:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Backup success rate',
+          targets=[t.backupSuccessRate],
+          description=|||
+            Success rate of backups.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percentunit')
+        + g.panel.timeSeries.standardOptions.withMin(0)
+        + g.panel.timeSeries.standardOptions.withMax(1),
+
+      backupSize:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Backup size',
+          targets=[t.backupSize],
+          description=|||
+            Size (in bytes) of backups for this cluster.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('decbytes'),
+
+      backupTime:
```

Comment:
This panel is a little difficult to read. It seems like the vertical axis doesn't mean anything, so it could be removed. The actual number that the color is changing based on doesn't have a unit, so it's hard to tell what the different colors mean. Let me know if there's anything I'm missing here.